dev-support/mini-submarine/conf/bootstrap.sh

```diff
@@ -36,6 +36,16 @@ for f in slaves core-site.xml hdfs-site.xml mapred-site.xml yarn-site.xml contai
   fi
 done
 
+# Copy submarine config
+for f in slaves submarine-site.xml submarine-env.sh; do
+  if [[ -e ${CONFIG_DIR}/$f ]]; then
+    cp ${CONFIG_DIR}/$f /opt/submarine-current/conf/$f
```

Comment:
`/opt/submarine-current/conf/`  already has latest submarine config, I think we could remove **submarine-site.xml** and **submarine-env.sh** in `dev-support/mini-submarine/conf/`, so when submarine config change we don't need to update config in mini-submarine as well

Reply:
@pingsutw, Thanks for the review.
/opt/submarine-current/conf/ has the latest submarine-site.xml and submarine-env.sh with default configuration. So we need to update them with the configuration suitable for mini-submarine.